activemq-client/src/main/java/org/apache/activemq/ActiveMQSession.java

```diff
@@ -2049,6 +2041,144 @@ protected void send(ActiveMQMessageProducer producer, ActiveMQDestination destin
         }
     }
 
+    /**
+     * Sends the message for dispatch by the broker.
+     *
+     * @param producer - message producer.
+     * @param destination - message destination.
+     * @param message - message to be sent.
+     * @param deliveryMode - JMS message delivery mode.
+     * @param priority - message priority.
+     * @param timeToLive - message expiration.
+     * @param disableTimestamp - disable timestamp.
+     * @param disableMessageID - optionally, disable messageID.
+     * @param producerWindow
+     * @param completionListener
+     * @param isProducerInCompletionListenerCallback
+     * @throws JMSException
+     */
+    protected void send(ActiveMQMessageProducer producer, ActiveMQDestination destination, Message message, int deliveryMode, int priority, long timeToLive,
+            boolean disableMessageID, boolean disableMessageTimestamp, MemoryUsage producerWindow, int sendTimeout,
+            CompletionListener completionListener, ThreadLocal<Boolean> isProducerInCompletionListenerCallback) throws JMSException {
```

Comment:
by java code convention, afaik, we shouldn't call fields and variables `isSomething", `is` is for methods that return boolean only

Reply:
ok will rename.